### PR TITLE
Update/docker config

### PR DESCRIPTION
**Pull request by mjfrancoeur**
Add working docker configuration for two services (database and server --> which will become the database API)

### CODE REVIEW

File: services/database/server.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const HOST = '0.0.0.0';
+
+// App
+const app = express();
+app.get('/', (req, res) => {
+  res.send('Hello world\n');
+});
+
+app.listen(PORT, HOST);
+console.log(`Running on http://${HOST}:${PORT}`);
```
Comment (TNaicker) [inline services/database/server.js]: I like the template literals. I think they make the code look cooler lol

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    environment:
+      - POSTGRES_USER=username
+      - POSTGRES_PASSWORD=password
+  db-services:
+    container_name: db-services
+    build: ./services/database
+    environment:
+      - DATABASE_URL=postgres://username:password@db:5432/users_db
+    links:
+      - db
```
Comment (TNaicker) [inline docker-compose.yml]: yml is for docker I'm assuming? Neat. This reminds me of webpack setup. Also the dockerfile

File: services/database/src/config/config.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-    "password": null,
-    "database": "testdatabase",
-    "host": "127.0.0.1",
+    "username": "username",
+    "password": "password",
+    "database": "text_desk_db",
+    "host": "db",
     "port": 5432,
     "dialect": "postgres"
   },
```
Comment (TNaicker) [inline services/database/src/config/config.json]: super secure ;]

File: services/database/src/config/config.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
 {
   "development": {
-    "username": "michaelfrancoeur",
-    "password": null,
-    "database": "testdatabase",
-    "host": "127.0.0.1",
+    "username": "username",
+    "password": "password",
```
Comment (TNaicker) [inline services/database/src/config/config.json]: super secure ;]

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    environment:
+      - POSTGRES_USER=username
+      - POSTGRES_PASSWORD=password
+  db-services:
+    container_name: db-services
+    build: ./services/database
+    environment:
+      - DATABASE_URL=postgres://username:password@db:5432/users_db
+      - POSTGRES_USER=username
+      - POSTGRES_PASSWORD=password
```
Comment (mjfrancoeur) [inline docker-compose.yml]: Update this later!

Comment (mjfrancoeur) [inline docker-compose.yml]: Update this later

**Pull request merged**